libstatistics_collector/test/topic_statistics_collector/test_received_message_age.cpp

```diff
@@ -97,8 +97,11 @@ TEST(ReceivedMessageAgeTest, TestAgeMeasurement) {
   EXPECT_TRUE(test_collector.IsStarted()) << "Expect to be started";
 
   rcl_time_point_value_t fake_now_nanos_{kStartTime};
+
   auto msg = DummyMessage{};
-  msg.header.stamp.nanosec = fake_now_nanos_;
+  msg.header.stamp.sec = static_cast<std::int32_t>(RCL_NS_TO_S(fake_now_nanos_));
```

Comment:
Is this field needed anywhere in the tests?

Reply:
It's used in the `OnMessageReceived` where the `DummyMessage` nanoseconds is reconstructed from the message header.